terraform/modules/services/nwp/secrets.tf

```diff
@@ -1,11 +1,7 @@
-# Read in secrets for API for MetOffice Weather DataHub
+# Read in required secrets for consumer
 
-data "aws_secretsmanager_secret" "nwp-api" {
+data "aws_secretsmanager_secret" "nwp-consumer-secret" {
   name = "${var.environment}/consumer/${var.consumer-name}"
   #  arn = "arn:aws:secretsmanager:eu-west-2::secret:development/consumer/nwp"
 }
 
-data "aws_secretsmanager_secret_version" "nwp-api-version" {
-  secret_id = data.aws_secretsmanager_secret.nwp-api.id
-  #  arn = "arn:aws:secretsmanager:eu-west-2::secret:development/consumer/nwp"
-}
```

Comment:
ah, perhaps this isnt need any more?

Reply:
It still is needed, I got confused about how to access the arn in the secret!